libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -57,7 +57,7 @@
   public MapboxTelemetry(Context context, String accessToken, String userAgent) {
     initializeContext(context);
```

Comment:
this constructor is rather heavy, do we really need to do all these things here?